editor/src/components/editor/editor-component.tsx

```diff
@@ -422,6 +427,22 @@ const useGithubData = (): void => {
     'Github data',
   )
 
+  // normalize github data
+  React.useEffect(() => {
+    dispatch([
+      EditorActions.updateGithubSettings({
```

Comment:
Can this normalisation be done in the actions? As currently this means a change to the dependency variables will cause this effect to fire and then dispatch a new action potentially triggering another re-render.

Reply:
good point yeah

Reply:
https://github.com/concrete-utopia/utopia/pull/2904/commits/ec20bdc04400531b53d6b8c9e586673cf6e01aa9